c++/src/capnp/layout.h

```diff
@@ -656,6 +654,8 @@ class ListReader {
   Data::Reader asData();
   // Reinterpret the list as a blob.  Throws an exception if the elements are not byte-sized.
 
+  Data::Reader asDataOfAnySize();
```

Comment:
Fine by me.  There's also getDataSectionAsBlob, which may make sense to rename (I haven't as of yet).